src/components/Footer.js

```diff
@@ -26,62 +28,124 @@ const socialMedias = [
 ];
 
 const { muiGridBlockContainer } = stylesBase;
-const useStyles = makeStyles({
+const useStyles = makeStyles(theme => ({
   muiGridBlockContainer,
   muiIconLogo: stylesBase.muiIconLogo,
-  lightIcon: { opacity: 0.6 },
+  subContainer: {
+    marginBottom: spacing(6),
+  },
+  menuContainer: {
+    [theme.breakpoints.down('xs')]: {
+      margin: 'auto',
+    },
+    [theme.breakpoints.down('sm')]: {
+      marginBottom: spacing(5),
+    },
+  },
+  lightIcon: ({ last }) => ({
+    opacity: 0.6,
+    marginRight: last ? '0px' : spacing(3),
+  }),
   itemContainer: {
     width: 'auto',
+    [theme.breakpoints.down('xs')]: {
+      marginBottom: spacing(4),
+    },
+  },
+  itemTextContainer: {
+    marginBottom: spacing(4),
   },
   itemText: {
     opacity: 1,
     fontWeight: 'bold',
-    marginBottom: spacing(4),
   },
   links: { textDecoration: 'none', marginBottom: spacing(4) },
-});
+}));
 
-const Footer = () => {
-  // eslint-disable-next-line no-shadow
-  const { muiGridBlockContainer, muiIconLogo, lightIcon, itemContainer, itemText, links } = useStyles();
+const Footer = ({ width }) => {
+  const {
+    // eslint-disable-next-line no-shadow
+    muiGridBlockContainer,
+    muiIconLogo,
+    subContainer,
+    menuContainer,
+    itemContainer,
+    itemTextContainer,
+    itemText,
+    links,
+  } = useStyles();
   return (
-    <Grid className={muiGridBlockContainer} container justify="flex-end">
-      <Grid item container xs={6} direction="column">
+    <Grid
+      className={muiGridBlockContainer}
+      container
+      justify="flex-end"
+      wrap={isWidthDown('sm', width) ? 'wrap-reverse' : 'wrap'}
+    >
+      <Grid
+        className={subContainer}
+        item
+        container
+        direction="column"
+        xs={isWidthDown('sm', width) ? 12 : 5}
+        alignItems={isWidthDown('sm', width) ? 'center' : 'flex-start'}
+      >
         <img className={muiIconLogo} src={logo} alt="Ovio" />
         <Grid>
-          {socialMedias.map(({ link, image: { imgSrc, imgAlt } }) => (
+          {socialMedias.map(({ link, image: { imgSrc, imgAlt } }, index) => (
             <a key={link} href={link} target="_blank" rel="noreferrer noopener">
-              <Icon component="img" className={lightIcon} src={imgSrc} alt={imgAlt} />
+              <Icon
+                component="img"
+                className={useStyles({ last: index === socialMedias.length - 1 }).lightIcon}
+                src={imgSrc}
+                alt={imgAlt}
+              />
             </a>
           ))}
         </Grid>
         <Typography variant="caption">© 2019 Ovio Inc - All Rights Reserved</Typography>
       </Grid>
-      <Grid item container xs={6} justify="space-between">
+      <Grid
+        item
+        container
+        xs={11}
+        sm={12}
```

Comment:
This might be totally fine, but I'm a little surprised to see the `sm` size wider than the `xs` one - is this on purpose?

Reply:
Oh yes, it is willful because from `sm` to `xs` the `flex-direction` is changing